docs/polkadot/implementors-guide/guide.md

```diff
@@ -1085,6 +1146,22 @@ enum CandidateBackingSubsystemMessage {
 }
 ```
 
+### Statement Gossip Subsystem Messages
+
+```rust
+enum StatementGossipSubsystemIn {
+  /// A statement to be gossiped out to validators.
+  StatementToGossip { relay_parent: Hash, statement: SignedStatement }
+}
+```
+
+```rust
+enum StatementGossipSubsystemOut {
+  /// A statement that we've received from a validator.
+  StatementReceived { relay_parent: Hash, statement: SignedStatement }
+}
+```
```

Comment:
Approval VRF gossiping can sign its gossip messages with their own VRFs.  Adds some code strangeness, but saves 64 bytes and 35ish us per gossip messages, maybe not worth the trouble.